plugins/wasm-go/extensions/ai-statistics/main.go

```diff
@@ -379,6 +430,16 @@ func getUsage(data []byte) (model string, inputTokenUsage int64, outputTokenUsag
 	return
 }
 
+func getUsage(data []byte) (model string, inputTokenUsage int64, outputTokenUsage int64, ok bool) {
+	// try to get the v1/completion
+	if model, inputToken, ouputToken, ok := getResponseTokenUsage(data); ok {
```

Comment:
It is suggested to determine the usage collection mode through the request path rather than falling back to chat completion